### ISSUE TITLE
BOSH cli show raw json data from api calls

### ISSUE THREAD

**Issue by warroyo** [cloudfoundry/bosh#1485]
currently with the bosh cli it does not seem possible to just retrieve the raw json output from the calls it is making to the bosh director. this makes it very hard to parse the data in scripts etc. Would  it make sense to add an option to output the raw json data from the api. Another possibility would be to add a `bosh curl` command similar to the `cf curl` command in the cloudfoundry cli that allows for making direct api call via the cli and leverages the cli's UAA authentication.

if something like this already exists please let me know.

**Comment by dpb587-pivotal** [comment]
We recently removed the `--raw` flag (it was broken). Can you describe a bit more about how you're using the raw output in your scripts? Perhaps we can find an alternative.

**Comment by dpb587-pivotal** [comment]
Oops, I misunderstood. We are currently working towards a new CLI written in go which is refactoring quite a few things. One of the new features is a `--json` flag which renders output in a script-friendly format. You might want to check out the work happening there (https://github.com/cloudfoundry/bosh-cli) and see if `--json` would be useful to you. The repository is still under active development and not yet stable, so things may end up changing (including the JSON output format) but it can at least give you an idea about where it's heading.

Another option may be for you to use the `director` library from that repository in your own go scripts - you can make the API calls directly and work with the responses in whatever way works best.

Do you think either of those will help in your automation needs?

**Comment by warroyo** [comment]
@dpb587 that is exactly what I am looking for, thank you for the information. I will keep an eye on the new cli.
